classes/form/cms_types_form.php

```diff
@@ -72,10 +72,10 @@ protected function definition() {
             get_string('mustache_template', 'cms')
         );
         $helptext = get_string('mustache_help', 'cms', $syntaxlink);
-        $helptext .= \html_writer::table($renderer->get_data_as_table());
+        $helptext .= \html_writer::table($renderer->get_data_as_table(true));
```

Comment:
For consistency can we have the `use html_writer;` defined up top? Same with `use moodle_url;` and then just use the `html_writer(), moodle_url()` without the leading backslashes

Reply:
done